<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackSettingsWidget</class>
 <widget class="QWidget" name="PlaybackSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="chromasoundRadioButton">
     <property name="text">
      <string>Chromasound</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chromasoundLabel">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>More compact.</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="standardRadioButton">
     <property name="text">
      <string>Standard</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="standardLabel">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>Compatible with all VGM players.</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="legacyRadioButton">
     <property name="text">
      <string>Legacy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="legacyLabel">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>Required for PCM on the Chromasound Nova Direct.</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
